examples/step-13/step-13.cc

```diff
@@ -1407,7 +1407,7 @@ namespace Step13
     GridGenerator::hyper_cube(triangulation, -1, 1);
     triangulation.refine_global(2);
     const FE_Q<dim>          fe(1);
-    const QGauss<dim>        quadrature(4);
+    const QGauss<dim>        quadrature(fe.degree + 1);
```

Comment:
This is not equal to the previous value 4. (The finite element is declared immediately above, having degree 1.) Have you verified that the output is the same?